lib/membrane_ice_plugin/ice_endpoint.ex

```diff
@@ -551,6 +551,12 @@ defmodule Membrane.ICE.Endpoint do
     {actions, state}
   end
 
+  @impl true
+  def handle_info(msg, _ctx, state) do
+    Membrane.Logger.error("Received unknown message: #{inspect(msg)}")
+    {[], state}
+  end
+
```

Comment:
```suggestion
```

Reply:
Can you justify that suggestion? This is needed to not crash webrtc endpoint in case of another unknown message.